rim/static/rim/js/base.js

```diff
@@ -1,12 +1,31 @@
 $(document).ready(function() {
-    $(document).foundation();
     function set_page_cookie(per_page) {
         var expire_date = new Date();
         //Sets the expire_date to be a year from now
         expire_date.setTime(expire_date.getTime() + (365*24*60*60*1000))
         document.cookie = "paginate=" + per_page +";expires=" + expire_date.toUTCString() +";path=/";
     }
-
+    
+    function hide_menu(){
+        //Obtain needed elements
+        let topnav = document.getElementById("topnav");
```

Comment:
is `let` used for a specific reason here? if not, previously existing code as well as other projects have used `var` instead. It would probably be good to stay consistent. Same applies to home.js

Reply:
Yes. While in this case it technically doesn't make a difference and "var" may be used, "let" is newer and provides better scoping capabilities. Keep in mind that rimproject is somewhat legacy code and while ES6 was released in the end of 2015 introducing "const" and "let", it was not widely used due to abundant ES5 code and compatibility issues with Safari, which is not a case anymore. I had a conversation with Shane about that and he suggested us to use "let" from now on. Although this doesn't mean that we need to change all "var"s to "let"s in our codebase because it gives us a visible borderline between newer and legacy code.